### PR TITLE
[ACR] Implement Eivor, Wolf-Kissed

### PR DESCRIPTION
**Pull request by theelk801**
Opening this as a PR in case anyone takes issue with the interface I added or anything else

### CODE REVIEW

**Comment by ssk97** [comment]
`CardTypeAssignment` is now just a strict subset of your `TypeAssignment`, right? Should probably just be one.

**Comment by xenohedron** [comment]
Have you seen `PredicateCardAssignment` and `TargetCardAndOrCard`? #11497 

I think it's best to use a common target class and not a custom target (for example, that includes the possibleTargets method which your implementation is missing).

I also think it's probably cleaner to just pass in predicates rather than add a new interface and method. Since predicates are more general and thus can be extended to things that aren't types (e.g. colors) as well as and/or/not combinations.

So basically, I think all you'd have to do is change a constructor from protected to private and use:
```java
new TargetCardAndOrCard(SubType.SAGA.getPredicate(),
        CardType.LAND.getPredicate(),
        "a Saga card and/or a land card")
```

**Comment by theelk801** [comment]
yeah I didn't realize that exists, I'll adapt it to use that
